packages/shadow-dom/src/findInsertionPoint.ts

```diff
@@ -39,3 +39,32 @@ export function findInsertionPoint(
 
   return null;
 }
+
+export function findShadowRootInsertionPoint(
```

Comment:
I decided that since we're really maintaining two arrays and `insertionPoint` is irrelevant to the `renderer` it was better so separate the find logic.

Assuming this approach gets a 👍 I'll add tests for this.

Reply:
@spmonahan I like the idea, I have pushed a small update to tests, so we don't need to repeat the logic ([3658c87](https://github.com/microsoft/griffel/pull/480/commits/3658c87d36df090c8987d848c65a257b65e87112)).

Reply:
Nice! Thanks for that!